gradle.properties.skeleton

```diff
@@ -0,0 +1,6 @@
+# these 2 properties are used only if one publishes snapshots to oss, see :uploadArchives gradle task
+ossUser=<<ask support of search respective wiki pages>>
```

Comment:
all of them are available in the encrypted files, besides `ossUser` and `ossKey` which are available on common _commercetools Accessing_ page, because publishing snapshots to OSS is not a part of common build  workflow but is used as temporary testing solution.